siliconcompiler/tools/openroad/scripts/sc_procs.tcl

```diff
@@ -110,3 +110,128 @@ proc sc_pin_placement { args } {
     {*}$openroad_ppl_arguments \
     {*}$ppl_args
 }
+
+###########################
+# Check if OR has a GUI
+###########################
+
+proc sc_has_gui {} {
+  if {[expr [llength [info procs save_image]] > 0]} {
```

Comment:
I know right now `save_image` and the SC GUI are pretty related.
Do you think it makes sense to have a more independent GUI flag in tcl?
Maybe in the future the SC user wants to create that image but is not using the GUI feature.

Reply:
`sc_has_gui` is just a helper function to check if OpenROAD was compiled with GUI support. I've just been prepending `sc_` to the helper functions to try and avoid name collisions

Reply:
Oh I thought this is referring to whether siliconcompiler's gui is turned on.
If it's about OpenROAD then it looks good to me.